_IBC/qwoyn-osmosis.json

```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "../ibc_data.schema.json",
+    "chain_1": {
+      "chain_name": "osmosis",
+      "client_id": "07-tendermint-6",
+      "connection_id": "connection-7"
+    },
+    "chain_2": {
+      "chain_name": "qwoyn",
+      "client_id": "07-tendermint-1",
+      "connection_id": "connection-0"
+    },
+    "channels": [
+      {
+        "chain_1": {
+          "channel_id": "channel-0",
```

Comment:
Do you have the channels backwards? Osmosis' channel-0 points to Cosmos Hub

Reply:
I think I did.